app/PhoneNumber.php

```diff
@@ -50,4 +71,15 @@ public static function findVerifiedByTwilioNumber($number)
 
         return self::where('number', $number)->where('is_verified', true)->firstOrFail();
     }
+
+    public function getStatusAttribute()
+    {
+        if ($this->blacklisted) {
```

Comment:
Same note as above RE: constant. This plus the verified scope may merit a trait?